ibis/backends/tests/test_window.py

```diff
@@ -1104,6 +1111,11 @@ def test_first_last(backend):
     reason="not support by the backend",
 )
 @pytest.mark.broken(["flink"], raises=Py4JJavaError, reason="bug in Flink")
+@pytest.mark.broken(
+    ["exasol"],
+    raises=ExaQueryError,
+    reason="database can't handle UTC timestamps in DataFrames",
```

Comment:
This might be fixable if we get rid of `import_from_pandas` and do the inserts ourselves.